site/signers/eoa.md

```diff
@@ -33,7 +33,7 @@ A JSON RPC based account is one where the key material is not available locally,
 ```ts
 import { createWalletClient, custom } from "viem";
 import { mainnet } from "@alchemy/aa-core";
-import { WalletClientSigner } from "@alchemy/core";
+import { WalletClientSigner } from "@alchemy/aa-core";
```

Comment:
thanks for the catch. you can join these lines to import both since it's the same package

Reply:
oops silly oversight